test/deprecated.lua

```diff
@@ -0,0 +1,9 @@
+-- Island of Misfit Toys
+
+local M = {}
+
+function M.redir_exec()
+  error('redir_exec is deprecated, use nvim_exec() or helpers.exec_capture()')
```

Comment:
For tests, I am not sure if this is really useful or if we should just remove it entirely. Trying it out anyway.